runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -731,11 +731,7 @@ local has_parser = function(lang)
     or #vim.api.nvim_get_runtime_file('parser/' .. lang .. '.*', false) > 0
 end
 
---- Return parser name for language (if exists) or filetype (if registered and exists)
----
----@param alias string language or filetype name
----@return string? # resolved parser name
-local function resolve_lang(alias)
+local function resolve_lang_helper(alias)
```

Comment:
can just inline this into the `resolve_lang` function? (removes indirection)

Reply:
sure, inlined and un-rolled (no for loop)